autonity/solidity/test/accountability.js

```diff
@@ -705,5 +710,265 @@ contract('Accountability', function (accounts) {
+      receipt = await utils.signAndSendTransaction(reporter, accountability.address, reporterPrivateKey, request);
+      assert.equal(receipt.status, true, "transaction failed");
+
+      currentEvent = await accountability.getReporterChunksMap({from: reporter});
+      hexProof = "0x" + rawProof[0].toString(16);
+      checkEvent(currentEvent, offender, reporter, event.chunkId, hexProof);
+    });
+
+    it("sends chunked events from multiple validator", async function () {
+      let reporter = [];
+      let reporterPrivateKey = [];
+      reporter.push(validators[0].nodeAddress);
+      reporter.push(validators[1].nodeAddress);
+      reporterPrivateKey.push(genesisPrivateKeys[0]);
+      reporterPrivateKey.push(genesisPrivateKeys[1]);
+
+      let offender = reporter;
+
+      let balance = web3.utils.toWei("10", "ether");
+      for (let i = 0; i < reporter.length; i++) {
+        await web3.eth.sendTransaction({from: validators[0].treasury, to: reporter[i], value: balance});
+      }
+
+      let PNrule = 0;
+      let event = {
+        "chunks": 3,
+        "chunkId": 0,
+        "eventType": 0,
+        "rule": PNrule,
+        "reporter": reporter,
+        "offender": offender,
+        "rawProof": [],
+        "block": 10,
+        "epoch": 0,
+        "reportingBlock": 11,
+        "messageHash": 0,
+      };
+
+      let currentProof = "0x";
+      let eventCount = event.chunks - 1;
+      for (let chunkId = 0; chunkId < eventCount; chunkId++) {
+        let rawProof = [];
+        rawProof.push(chunkId);
+        event.chunkId = chunkId;
+        event.rawProof = rawProof;
+        let hexNumber = (chunkId > 15) ? chunkId.toString(16) : "0" + chunkId.toString(16);
+        currentProof = currentProof + hexNumber;
+        // reporter[i] sends event for offender[i];
+        for (let i = 0; i < reporter.length; i++) {
+          let sender = reporter[i];
+          event.reporter = sender;
+          event.offender = offender[i];
+          let request = (await accountability.handleEvent.request(event, {from: sender}));
+          let receipt = await utils.signAndSendTransaction(sender, accountability.address, reporterPrivateKey[i], request);
+          assert.equal(receipt.status, true, "transaction failed");
+          let currentEvent = await accountability.getReporterChunksMap({from: sender});
+          checkEvent(currentEvent, offender[i], sender, chunkId, currentProof);
+        }
+      }
+
+      event.chunkId = eventCount;
+      for (let i = 0; i < reporter.length; i++) {
+        event.reporter = reporter[i];
+        event.offender = offender[i];
+        await truffleAssert.fails(
+          accountability.handleEvent.call(event, {from: reporter[i]}),
+          truffleAssert.ErrorType.REVERT,
+          "failed proof verification"
+        );
+      }
+
+    });
+
+    it.skip("has proper value in fields in storage", async function () {
```

Comment:
```suggestion
    it.skip("can start with chunkId = 1", async function () {
```

This name is more representative, unless I am missing something?